.github/workflows/build-debug.yml

```diff
@@ -18,13 +18,15 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '17'
-          cache: 'gradle'
 
-      - name: Build project with Gradle
-        run: chmod +x ./gradlew && ./gradlew app:assembleDebug --no-daemon
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
 
-      - uses: actions/upload-artifact@v3
+      - name: Build project
+        run: ./gradlew assembleDebug
+
+      - uses: kittaakos/upload-artifact-as-is@v0
```

Comment:
> i mean that repo is dead and i dont see much of a point in this either whats wrong with the upload-artifact?

Regular one uploads them all named artifact instead of using the filename